app/src/main/res/values/strings.xml

```diff
@@ -708,7 +708,7 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
 
     <string name="quest_building_entrance_title">What kind of building entrance is this?</string>
     <string name="quest_building_entrance_main">Main entrance of a public building</string>
-    <string name="quest_building_entrance_staircase">Staircase entrance (e.g. of apartment building)</string>
+    <string name="quest_building_entrance_staircase">Stairwell entrance (e.g. to an apartment building)</string>
```

Comment:
According to merriam-webster:

> **staircase** 
> 1 : the structure containing a [stairway](https://www.merriam-webster.com/dictionary/stairway)
> 2 : a flight of [stairs](https://www.merriam-webster.com/dictionary/stairs) with the supporting framework, casing, and balusters

> **stairwell**
> a vertical shaft in which [stairs](https://www.merriam-webster.com/dictionary/stairs) are located

We are after meaning 1 of staircase here.

Reply:
Yeah that sounds right to me.  There is also stair_way_... :-D

Reply:
So should it be changed back?

Reply:
The `an` is needed.  Either choice of stairwell or staircase would be fine in en_US.  I think the changed version is slightly nicer if you're describing an apartment building.